test/api/examples/programsMockData.ts

```diff
@@ -0,0 +1,38 @@
+import { Programs } from '@prisma/client';
+import { ProgramsWrite } from '../../../lib/types/dtos/createProgramsRequest.dto';
+
+export const createPrograms0: ProgramsWrite = {
```

Comment:
💭 I think we should clearly separate good examples and bad examples within this file.